@Vipul-Cariappa, @aaronj0, this change is a net win -- it enables 35 new tests or so at the cost of 1 failing. Can you take care of this and adjust the tests on osx correspondingly:
```
2026-05-08T19:47:18.4904950Z FAILED test_cpp11features.py::TestCPP11FEATURES::test16_unique_ptr_moves - TypeError: Could not find "make_unique<int>" (set cppyy.set_debug() for C++ errors):
2026-05-08T19:47:18.4905040Z   std::unique_ptr<int> std::make_unique<int, 0>() =>
2026-05-08T19:47:18.4905120Z     TypeError: takes at most 0 arguments (1 given)
2026-05-08T19:47:18.4905220Z   none of the 2 overloaded methods succeeded. Full details:
2026-05-08T19:47:18.4905360Z   std::unique_ptr<int> std::make_unique<int, int, 0>(int && __args) =>
2026-05-08T19:47:18.4905450Z     ValueError: nullptr result where temporary expected
2026-05-08T19:47:18.4905540Z   std::unique_ptr<int> std::make_unique<int, 0>() =>
2026-05-08T19:47:18.4905620Z     TypeError: takes at most 0 arguments (1 given)
2026-05-08T19:47:18.4905770Z = 1 failed, 393 passed, 48 skipped, 92 xfailed, 35 xpassed, 101 warnings in 16.83s =
```

--- a/lib/CppInterOp/CppInterOp.cpp
+++ b/lib/CppInterOp/CppInterOp.cpp
@@ -3851,11 +3851,9 @@ bool DefineAbsoluteSymbol(compat::Interpreter& I, const char* unmangled_name,
 
   // mangleAndIntern applies the target DataLayout's symbol prefix
   // (leading `_` on Mach-O, no-op on ELF) so the registered key
-  // matches what the JIT computes when it lowers an IR-level symbol
+  // matches what the JIT computes when it lowers an IR symbol
   // reference for lookup. Plain ES.intern() bypasses the prefix and
-  // silently breaks lookup on Mach-O — e.g.
-  // `__clang_Interpreter_SetValueNoAlloc` registered, but the JIT
-  // looks up `___clang_Interpreter_SetValueNoAlloc`.
+  // silently breaks lookup on Mach-O.
   llvm::orc::SymbolMap InjectedSymbols{
       {Jit.mangleAndIntern(unmangled_name),
        ExecutorSymbolDef(ExecutorAddr(address), JITSymbolFlags::Exported)}};

